### DISCUSSION TITLE
Q-Q plot

### DISCUSSION THREAD

**Discussion by tatiana-pashkova** [RePsychLing/SMLP2023#29] [Q&A] [answered]
In the course, we've been looking at Q-Q plots and we seemed to concentrate on whether the dots are on the diagonal line, or if they fan out, or if they form a curve. I am wondering about the line itself - does it have to be strictly at 45 degrees or is a smaller angle fine too?

**Comment by palday** [comment, marked as answer]
Ideally if we made all the Q-Q plots perfectly, they would have a 1:1 aspect ratio (i.e. be perfectly square). In that case, the ideal line would be at a 45 degree angle across the plot. Unfortunately, the aspect ratio isn't always correct (I'm working on it!), so the ideal line would differ from the 45 degree angle.

That said, the plotted line is by default a robust fit line, i.e. an attempt to find the _observed_ correspondence between the theoretical and observed quantiles. So if it's differing from the ideal line, then that in itself is an indication that we're not achieving normality. The observed points may differ from the line a bit, but it's most telling when they differ on the ends because it means that the tails are too heavy (or in rare cases too light), even if we're being generous about the correspondence.

You can change the type of line that's displayed by the way:

https://docs.makie.org/stable/reference/plots/qqplot/

https://palday.github.io/MixedModelsMakie.jl/stable/api/#QQ-Plots

Maybe I should change the default to be `:identity` (so the line of ideal correspondence) in the next breaking release. :thinking:
